conf/data/product-draft.json

```diff
@@ -0,0 +1,240 @@
+{
+  "name": {
+    "en": "Classic box"
+  },
+  "description": {
+    "en": "Box with classic flavours."
+  },
+  "categories": [],
+  "categoryOrderHints": {},
+  "slug": {
+    "en": "donut-box"
+  },
+  "masterVariant": {
+    "sku": "box6",
+    "prices": [
+      {
+        "value": {
+          "currencyCode": "EUR",
+          "amount": 6.99
+        }
+      }
+    ],
+    "images": [
+      {
+        "url": "https://d7425bc8dad60afc1fb6-19c25edf7c08d49ff8f45ca05592de1c.ssl.cf3.rackcdn.com/donuts-WuuDavUS.png",
+        "dimensions": {
+          "w": 264,
+          "h": 190
+        }
+      },
+      {
+        "url": "https://d7425bc8dad60afc1fb6-19c25edf7c08d49ff8f45ca05592de1c.ssl.cf3.rackcdn.com/stamp-green-C3OqK9-I.png",
+        "dimensions": {
+          "w": 404,
+          "h": 800
+        }
+      },
+      {
+        "url": "https://d7425bc8dad60afc1fb6-19c25edf7c08d49ff8f45ca05592de1c.ssl.cf3.rackcdn.com/save-green-AfNyfCRG.png",
+        "dimensions": {
+          "w": 213,
+          "h": 100
+        }
+      }
+    ],
+    "attributes": [
+      {
+        "name": "quantity",
+        "value": 6
+      },
+      {
+        "name": "box",
+        "value": "Small box"
+      },
+      {
+        "name": "pactas1",
+        "value": "5231dfec1d8dd00f389dc547"
```

Comment:
All these Pactas IDs will only work with our Pactas account, which of course nobody else will have. We should let the developer provide these instead of giving them via the import. Maybe we can provide them via configuration file?
In any case, create better a ticket to improve import and let's solve this in another PR.